src/IntegratedExperts/BehatScreenshot/Context/Initializer/ScreenshotContextInitializer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @file
+ * This file is part of the IntegratedExperts\BehatScreenshot package.
+ */
+
+namespace IntegratedExperts\BehatScreenshot\Context\Initializer;
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\Initializer\ContextInitializer;
+use IntegratedExperts\BehatScreenshot\Context\ScreenshotContextInterface;
+
+/**
+ * Class ScreenshotContextInitializer
+ */
+class ScreenshotContextInitializer implements ContextInitializer
+{
+    /**
+     * Context parameters.
+     *
+     * @var array
+     */
+    private $parameters;
+
+    /**
+     * Does need to clear directory trigger.
+     *
+     * @var bool
+     */
+    private $purge = true;
+
+    /**
+     * ScreenshotContextInitializer constructor.
+     *
+     * @param array $parameters
+     */
+    public function __construct($parameters)
+    {
+        $this->parameters = $parameters;
```

Comment:
please remove parameters and use protedted variables (such as `purge`) and resolve them in the constructor